api-scripts/inventory-storage/inventoryStorage_db_pool_utilization/jmeter-supported-data/credentials_[tenant].csv

```diff
@@ -0,0 +1,2 @@
+username,password,tenant,serverName
+[username],[password],[tenantId],[serverName]
```

Comment:
How about renaming [serverName] to [targetURL] for it to be consistent with the current carrier-io Jenkins job parameter?